docs/configuring-swc.md

```diff
@@ -118,8 +127,6 @@ Starting from `@swc/core` v1.1.4, you can enable loose mode by
 }
 ```
 
-In loose mode, swc generates more efficient code.
```

Comment:
I removed this line since it is not recommended. Let me know if you want to add it back.

Reply:
I'm fine with it. Thanks!